File: algorithms/insertion_sort/insertionSort.js
function insertionSort(arr) {
  let j = 0
  for (let i = 1; i < arr.length; i++) {
    j = i

    while (j > 0 && arr[j - 1] > arr[j]) {
      let tmp = arr[j]
      arr[j] = arr[j - 1]
      arr[j - 1] = tmp
      j--
    }
  }

  return arr
}

File: algorithms/insertion_sort/insertionSort.py
def insertionSort(arr):
    n = len(arr)
    i = 1

    while i < n:
        j = i
        i += 1
        while j > 0 and arr[j-1] > arr[j]:
            arr[j-1], arr[j] = arr[j], arr[j-1]
            j -= 1
    return arr